src/test/resources/testnextflowvdsl3/workflows/pipeline3/main.nf

```diff
@@ -1,12 +1,11 @@
 nextflow.enable.dsl=2
 
-include { readConfig; helpMessage; preprocessInputs; channelFromParams } from "${params.rootDir}/workflows/utils/WorkflowHelper.nf"
+include { readConfig; helpMessage; channelFromParams } from "${params.rootDir}/workflows/utils/WorkflowHelper.nf"
 config = readConfig("${params.rootDir}/workflows/pipeline3/config.vsh.yaml")
 
 workflow base {
   helpMessage(config)
 
   channelFromParams(params, config)
-    | preprocessInputs(["config": config])
     | view {"DEBUG: $it"}
 }
```

Comment:
feel free to delete this file -- cfr our discussion

Reply:
Done!